websites/A/Animeflix/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "326694667003363328",
+		"name": "Micotsu#2907"
```

Discriminator number is a mistake caused by an old version of `pmd`
```suggestion
		"name": "Micotsu"
```